histomicstk/segmentation/label/trace_boundary.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import ctypes
-from .isbf import isbf
+from .trace_object_boundary import trace_object_boundary
 
 
 def trace_boundary(Mask, Connectivity=4, XStart=None, YStart=None,
```

Comment:
@cdeepakroy @slee172 I think this is a good suggestion. Merge the functions and add a check to detect whether an image is label or binary.